src/todo/controllers/todo.controller.ts

```diff
@@ -8,7 +8,8 @@ import {
   Post,
   Put,
   Body,
-  Delete
+  Delete,
+  Query
```

Comment:
The `findAll` method in the `TodoController` now accepts query parameters for filtering todos by title and completion status. Ensure that the `completed` query parameter is properly validated and sanitized to prevent potential security issues, such as injection attacks.